config/environments/production.rb

```diff
@@ -49,14 +61,15 @@
   config.log_level = :info
 
   # Prepend all log lines with the following tags.
-  # config.log_tags = [ :subdomain, :uuid ]
-
-  # Use a different logger for distributed setups.
-  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
+  config.log_tags = [ :request_id ]
```

Comment:
I assume this will change how our logs look in Splunk. I think we should take a note to take a look once this hits production and see if we need to adjust.